pyke/kepsff.py

```diff
@@ -0,0 +1,760 @@
+        colored red.
+    npoly_arfl : integer
+        The fourth panel (lower-middle) provides the subsequent target motion
+        vs aperture flux distribution. The red line is polynomial fit, the
+        order of which is chosen by the user here. Iterative σ-clipping is
+        again required - the many outliers below the best-fit are astrophysical
+        in nature (binary star eclipses) and would bias the fit if they were
+        unrecognized or simply allowed to. The user has flexibility to change
+        the order and clipping threshold in order to provide the most robust
+        fit and there is a degree of objectivity in this process. A low
+        polynomial is generally adequate, this example employs a 3rd order
+        function.
+    sigma_arfl : float [sigma]
+        This is the threshold limit in units of 1-sigma standard deviation from
+        the best-fit polynomial to the motion-flux relation. Data points
+        falling outside of the threshold are in effect rejected from the
+        calibration. Such points potentially contain astrophysical signal that
+        would both bias the calculation and damp the astrophysical signal
+        within the data after the correction has been made. The best-fit red
+        line provides the correction factors within the data window. For each
+        point along the aperture-derived flux curve (upper-right), the position
+        along the arc can be calculated and the multiplicative correction
+        factor to the flux can determined directly from the best-fit
+        motion-flux relation. The subsequent corrected light curve is provided
+        lower-right on the same plotting scale as above. The red points are
+        those flagged as potential thruster firing and their 6-hr cadence
+        suggests these events have generally been flagged well.
+    plotres : boolean (optional)
+        If true, diagnostic plots identical to figure 1 above will be rendered
+        and also saved as PNG files. There will be a different plot for every
+        time window defined by the stepsize parameters. If the output FITS file
+        is named filename.fits then each PNG file will be named
+        filename_nn.png, where nn is a sequential number beginning with 1.
+    clobber : bool (optional)
+        Overwrite the output FITS file? if clobber = no and an existing file
+        has the same name as outfile then the task will stop with an error.
+    verbose : bool (optional)
+        Print informative messages and warnings to the shell and logfile?
+    logfile = string (optional)
+        Name of the logfile containing error and warning messages.
+    """
+    # log the call
+    hashline = '--------------------------------------------------------------'
+    kepmsg.log(logfile,hashline,verbose)
+    call = ('KEPSFF -- '
+            'infile={}'.format(infile)
+            'outfile={}'.format(outfile)
```

Comment:
I'm surprised this works for you -- it gives a SyntaxError on Python 3.5 for me. (Which I can fix by adding a "+" in front if each line.